.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,9 +1,10 @@
 ---
 name: Bug report
 about: Filling a bug report
-title: ""
+title: ''
 labels: bug
-assignees: ""
+assignees: ''
```

Comment:
These changes came in from GitHub's issue template web ui. It looks fine to me.

Reply:
could you clarify? 
I do see the change to single-quote in git, are you referring to something else?

Reply:
Nope, that was it. Just the single-quote change that GitHub did 😄

Reply:
ohhh I see, the changes were done by GH, I'd misunderstood 👍